### PR TITLE
Feature/account settings edit cc subscription info

### PR DESCRIPTION
**Pull request by RyanMersmann**
# Description

Fixes #

- What work was done? adding the account settings buttons to the account settings component and making sure that they unlocked with the rest of the settings.
- Why was this work done? To give the parent the option to edit their cc info or subscription plan
- What feature / user story is it for? As a parent, I would like a clear parent dashboard that allows for changing my subscription settings/ credit card info
- Any relevant links or images / screenshots
- 
image

## Type of change

Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)

## Change Status
- [x] Complete, tested, ready to review and merge

## Has This Been Tested
- [x] Yes

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/pages/AccountSettings/RenderAccountSettings.js [hunk 1]
```diff
@@ -116,6 +116,10 @@ function RenderAccountSettings() {
         style={unlock ? { opacity: '.3' } : null}
       >
         <AccountSettingsForm disabled={unlock} />
+        <div classname="settings-buttons-cotainer">
+          <Button disabled={unlock}>Edit Credit Card Info</Button>
+          <Button disabled={unlock}>Edit Subscription Plan</Button>
+        </div>
       </div>
     </div>
   );
```
Comment (Robert-Lark) [inline src/components/pages/AccountSettings/RenderAccountSettings.js:125]: Nice work - if you look in the less for the accountSettings page you should be able to position these pretty easily using the css grid that is overlaid over the entire component - let me know if you haven't used grid before and I can run through it with you - super simple, very semantic also so you should be able to see how it works just by looking at what rules are applied to the other button.

File: src/styles/less/AccountSettings.less [hunk 2]
```diff
@@ -48,7 +48,9 @@
         background-color: #8ce2e0;
     }
 }
-
+.settings-buttons-cotainer {
+  border: 1px solid red;
+}
```
Comment (Robert-Lark) [inline src/styles/less/AccountSettings.less:54]: I imagine this was just for dev purposes and will be removed at a later time?

**Comment by RyanMersmann** [comment]
as a note, I did an npm audit fix so that is why it says the package-lock.json is different

**Review by Robert-Lark — DISMISSED** [review]
Nice work

File: src/styles/less/AccountSettings.less (hunk 2 above)
Comment (RyanMersmann) [reply src/styles/less/AccountSettings.less:54]: Yes, it will be removed when I add the proper styling. it's not visible on the page.

**Comment by Robert-Lark** [comment]
> as a note, I did an npm audit fix so that is why it says the package-lock.json is different

ah - yea I figured - that was the first thing I noticed that it said +26000 lines of code - I was like - wow, these guys have been busy haha

**Comment by MosharrafMusa** [comment]
looks great!

**Review by StefxM — DISMISSED** [review]
Loving this!

**Review by MosharrafMusa — APPROVED** [review]
looks good!

**Review by StefxM — APPROVED** [review]
looks good !

**Pull request merged**